tool/tctl/common/resource_command.go

```diff
@@ -215,10 +216,15 @@ func (rc *ResourceCommand) GetAll(client auth.ClientI) error {
 }
 
 // Create updates or inserts one or many resources
-func (rc *ResourceCommand) Create(client auth.ClientI) error {
-	reader, err := utils.OpenFile(rc.filename)
-	if err != nil {
-		return trace.Wrap(err)
+func (rc *ResourceCommand) Create(client auth.ClientI) (err error) {
+	var reader io.Reader
+	if rc.filename == "" {
```

Comment:
Isn't `-` used to represent stdin?

Reply:
See my [comment](https://github.com/gravitational/teleport/pull/4906#discussion_r526447847) above